packages/desktop-client/src/components/reports/DateRange.tsx

```diff
@@ -72,9 +72,9 @@ export function DateRange({ start, end, type }: DateRangeProps): ReactElement {
             Compare {{ formattedStartDate }} to {{ typeOrFormattedEndDate }}
           </Trans>
         ) : (
-          <Trans>
-            {{ formattedStartDate }} - {{ formattedEndDate }}
-          </Trans>
+          <>
+            {formattedStartDate} - {typeOrFormattedEndDate}
```

Comment:
To be a little more explicit, since `type` can't occur here:
```suggestion
            {formattedStartDate} - {formattedEndDate}
```

Reply:
I think this was an autocorrect mistake, fixed, thank you